project2/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Rubik:400,500&display=swap" rel="stylesheet">
+    <title>Movie Card</title>
+</head>
+
+<body>
+    <div>
+        <!--image was found on Google-->
+        <img src="images/YTuMamaTambien.jpg">
+        <section class="info">
+            <h2>Y Tu Mamá También</h2>
+            <div class="stars">
+                <svg width="24" height="24" viewBox="0 0 24 24">
+                    <path
+                        d="M12,17.27L18.18,21L16.54,13.97L22,9.24L14.81,8.62L12,2L9.19,8.62L2,9.24L7.45,13.97L5.82,21L12,17.27Z">
+                    </path>
+                </svg>
+                <svg width="24" height="24" viewBox="0 0 24 24">
+                    <path
+                        d="M12,17.27L18.18,21L16.54,13.97L22,9.24L14.81,8.62L12,2L9.19,8.62L2,9.24L7.45,13.97L5.82,21L12,17.27Z">
+                    </path>
+                </svg>
+                <svg width="24" height="24" viewBox="0 0 24 24">
+                    <path
+                        d="M12,17.27L18.18,21L16.54,13.97L22,9.24L14.81,8.62L12,2L9.19,8.62L2,9.24L7.45,13.97L5.82,21L12,17.27Z">
+                    </path>
+                </svg>
+                <svg width="24" height="24" viewBox="0 0 24 24">
+                    <path
+                        d="M12,17.27L18.18,21L16.54,13.97L22,9.24L14.81,8.62L12,2L9.19,8.62L2,9.24L7.45,13.97L5.82,21L12,17.27Z">
+                    </path>
+                </svg>
+                <svg width="24" height="24" viewBox="0 0 24 24">
+                    <path
+                        d="M12,17.27L18.18,21L16.54,13.97L22,9.24L14.81,8.62L12,2L9.19,8.62L2,9.24L7.45,13.97L5.82,21L12,17.27Z">
+                    </path>
+                </svg>
+            </div>
+            <div>
+                <a class="play" href="https://www.youtube.com/watch?v=UW1LQbtRuAM">
+                    <svg class="playbutton" xmlns="http://www.w3.org/2000/svg" viewBox="27 23 46 54">
+                        <path d="M29 23h1l42 25v4L30 77c-1 1-3 0-3-2V25l1-2h1z" />
+                    </svg>
+                    play
+                </a>
+            </div>
+            <p>In Mexico, two teenage boys and an attractive older woman embark on a road trip and
+                learn a thing or two about life, friendship, sex, and each other.</p>
+            <svg class="bookmark" width="24" height="24" viewBox="0 0 24 24">
```

Comment:
This SVG would need to be wrapped in either a link or a button element to be made functional (my guess is that a button is more appropriate if the action happens on this page and doesn't open up a new link).